### ISSUE TITLE
Transformer tuning for low-resource NMT

### ISSUE THREAD

**Issue by fairwell-coding** [fairwell-coding/nlp#4]
Use approaches described in 

- https://aclanthology.org/2021.mtsummit-research.5/
- https://aclanthology.org/2020.coling-main.304/

in order to finetune the architecture and also training approach (label smoothing, # attention heads, ... etc). The first one is more about hyperparameters for training and that it indeed performs well. The second paper precisely has some good architecture approaches which shall be applied on the already distilled version. 

Verify that the changes work (locally via model inference and starting training shortly just for confirmation). Add everything need for later training, i.e. checkpoint saving/loading etc ... after this task has been resolved one should be able to simply start finetuning of this low-resource model.
